.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,62 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+<!--
+Thank you for contributing to the Dataverse Project!
```

Comment:
Thank you for contributing to the Dataverse Project through the creation of a bug report!